packages/common/src/intl/intlHelpers.ts

```diff
@@ -48,7 +48,9 @@ export const useCurrentLanguage = (): SupportedLocales => {
 
 export const useFormatDate = (): ((
   value: number | Date,
-  options?: Intl.DateTimeFormatOptions & { format?: string }
+  options?: Intl.DateTimeFormatOptions & { format?: string } & {
```

Comment:
just so I could tweak the formatting.. this is the shape that the underlying intl uses